idea/idea-core/src/org/jetbrains/kotlin/idea/core/ExpectedInfos.kt

```diff
@@ -666,10 +666,10 @@ class ExpectedInfos(
                 val (setValueOperator, setOperatorSubstitutor) = typesWithSetDetector.findOperator(substitutedType) ?: return null
                 val propertyType = explicitPropertyType ?: getValueOperator.fuzzyReturnType()!!
                 val setParamType = setValueOperator.valueParameters.last().type.toFuzzyType(setValueOperator.typeParameters)
-                val setParamTypeSubstitutor = setParamType.checkIsSuperTypeOf(propertyType) ?: return null
+                val settypeParameterSubstitutor = setParamType.checkIsSuperTypeOf(propertyType) ?: return null
```

Comment:
Capitalize `t` in `type`

Reply:
Oops! Actually I shouldn't change this file at all!